src/test/java/com/ericsson/gerrit/plugins/eiffel/handlers/DataBaseHandlerTest.java

```diff
@@ -0,0 +1,208 @@
+package com.ericsson.gerrit.plugins.eiffel.handlers;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.powermock.api.mockito.PowerMockito.mock;
+
+import java.io.File;
+import java.net.ConnectException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.UUID;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.mockito.BDDMockito;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.ericsson.gerrit.plugins.eiffel.handlers.DataBaseHandler.Table;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(fullyQualifiedNames = "com.ericsson.gerrit.plugins.eiffel.*")
+public class DataBaseHandlerTest {
+    private final String branch = "my_test_branch";
+    private final String scmTableKey = "branch";
+    private final String scmpTableKey = "changeId";
+
+    private DataBaseHandler dbHandler;
+
+    @Rule
+    public TemporaryFolder testFolder = new TemporaryFolder();
+
+    @Before
+    public void init() throws Exception {
+        File tmpFolderPath = testFolder.newFolder();
+        dbHandler = new DataBaseHandler(tmpFolderPath, "test_file_name.db");
+    }
+
+    /**
+     * An exception should be raised if connection to Db cannot be established.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testFailingToConnectToDbThrowsError() throws Exception {
+        try {
```

Comment:
Is there a reason not specify the expected exception in the annotation as described on https://github.com/junit-team/junit4/wiki/exception-testing?

Reply:
changed...